### ISSUE TITLE
8.0.0 - Minimize / Minify / Uglify issue - table does not work

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/vue-handsontable-official#188]
@AndreyKomin commented on [Thu Aug 20 2020](https://github.com/handsontable/handsontable/issues/7207)

### Description
The table stops working if I build it in Production mode within a vue-cli project.
It seems a minification issue - if I disable `minimize` option in `vue.config.js` then it works.
![image](https://user-images.githubusercontent.com/7191525/90779690-cc975480-e306-11ea-99cf-76c32fe9ab1f.png)

It worked correctly in the 7.4.2 ver.

### Steps to reproduce

1. Unzip the `Archive.zip` (attached)
2. `npm install`
3. `npm run build && npm run start` (http://localhost:8080)
then you'll see this error:
![image](https://user-images.githubusercontent.com/7191525/90778003-c7d1a100-e304-11ea-92ff-e9986b5459f5.png)

Note: If you run it in a dev mode it would work (`npm run serve`)

### Demo
[Archive.zip](https://github.com/handsontable/handsontable/files/5103487/Archive.zip)

### Your environment
* Handsontable version: 8.0.0
* handsontable/vue version: 5.0.1
* Browser Name and version: Chrome 84
* Operating System: macOS Catalina

---

@AndreyKomin commented on [Sun Aug 23 2020](https://github.com/handsontable/handsontable/issues/7207#issuecomment-678757254)

@budnix @AMBudnik
Seems critical because vue users can't use 8.0.0 in production

---

@AMBudnik commented on [Mon Aug 24 2020](https://github.com/handsontable/handsontable/issues/7207#issuecomment-679010483)

@AndreyKomin 

Sorry for keeping you waiting. I have replicated the error in the console on my Windows 10.

I've removed Handsontable and the wrapper from the dependency tree and reloaded it back again (using HOT 8.0.0 / Vue wrapper 5.0.1) but got the same issue. Once I loaded HOT 7.4.2 and Vue wrapper 4.1.1 problem was gone.

One thing remains a mystery though. I wasn't able to find the `minimize` option in the project.

**Comment by AndreyKomin** [comment]
@AMBudnik 
Just create `vue.config.js` in the root with the following content:
```
module.exports = {
  configureWebpack: {
    optimization: {
      minimize: true
    }
  }
}
```
minimize: true - error
minimize: false - no error

To check it you should run `npm run build && npm run start`

You can read more about webpack build customization within a vue-cli project:
https://cli.vuejs.org/config/#global-cli-config

**Comment by AMBudnik** [comment]
> minimize: true - error
> minimize: false - no error

True. I also confirm. 
I wasn't aware that this option is true by default. Thank you for noticing and sharing this issue report.

**Comment by AndreyKomin** [comment]
@AMBudnik @budnix 
Any updates?
Maybe you can try to explain what's you found or some details and then I'll try to fix?

**Comment by AMBudnik** [comment]
@AndreyKomin our developer is already notified about this case, but he has a queue of tasks that he needs to accomplish first.

**Comment by AndreyKomin** [comment]
@AMBudnik @budnix
Any chance to know an approximate bugfix date?

**Comment by AMBudnik** [comment]
Hi @AndreyKomin we did not set a Roadmap for Vue yet. But we should have an update soon.

Inform [forum](https://forum.handsontable.com/t/vue-188-handontable-vue-error-when-running-in-production/4743/3)

**Comment by aninde** [comment]
I'm glad to see that this issue will be fixed on the upcoming release (current `develop`). I tested it locally with `Vue-CLI`.
I had a different reproduction, I entered `npm run serve` instead of  `start`.

If all goes well we plan to release a new version tomorrow.

**Comment by AMBudnik** [comment]
I'm more than happy to see this issue solved for v [5.1.0](https://github.com/handsontable/vue-handsontable-official/releases/tag/5.1.0)
@AndreyKomin please let me know if it works well for you after updating.
